.gitignore

```diff
@@ -7,3 +7,6 @@
 /.tox
 /dist
 /venv*
+.python-version
+.idea/*
+htmlcov/*
```

Comment:
please don't touch gitignore, thanks

Reply:
fair enough, I guess I'll move stuff I want to ignore specific to my setup to the [`.git/info/exclude`](https://git-scm.com/docs/gitignore#_description) file.